src/pages/RegistrationPage/RegistrationPage.js

```diff
@@ -50,6 +50,8 @@ const RegistrationPage = () => {
     const haveErrors = [];
     haveErrors.push(validator.validateField(firstName.input, setFirstName, validator.validateRequiredField));
     haveErrors.push(validator.validateField(lastName.input, setLastName, validator.validateRequiredField));
+    haveErrors.push(validator.validateField(firstName.input, setFirstName, validator.validateFieldLength));
```

Comment:
Why in one function you just check if text in field exist and in another if its length is correct, I think it would be enough to call just one function for that :)